wandb/integration/openai/fine_tuning.py

```diff
@@ -313,6 +307,18 @@ def _unpack_hyperparameters(cls, hyperparameters: Hyperparameters):
             return None
 
         return hyperparams
+    
+    @staticmethod
+    def sanitize(input: Any) -> dict | list | str:
```

Comment:
One small nit on type annotation. Across the wandb repo, I have seen the usage of `typing` module for type annotation. Not sure, if we can start using | for typing? cc: @kptkin

Reply:
yeah, we need to support python 3.7 and this notion was only introduced in python 3.10. see here: https://docs.python.org/3/library/stdtypes.html#union-type

Reply:
(i think you can use `from __future__ import annotations`), but maybe it is easier to be consistent with the rest of the repo and we will update the types in a future refactor.

Reply:
thanks. changed `dict | list | str` -> `Union[Dict, List, str]`